src/components/Button/index.tsx

```diff
@@ -253,11 +253,15 @@ const ButtonErrorStyle = styled(Base)`
   }
 `
 
-export function ButtonConfirmed({ confirmed, ...rest }: { confirmed?: boolean } & ButtonProps) {
+export function ButtonConfirmed({
+  confirmed,
+  altDisbaledStyle,
+  ...rest
+}: { confirmed?: boolean; altDisbaledStyle?: boolean } & ButtonProps) {
   if (confirmed) {
     return <ButtonConfirmedStyle {...rest} />
   } else {
-    return <ButtonPrimary {...rest} />
+    return <ButtonPrimary {...rest} altDisbaledStyle={altDisbaledStyle} />
```

Comment:
typo

Reply:
also don't think this is doing anything

Reply:
fixed the typo everywhere - we just need that to keep it a solid color background but also have it be in the disabled state (not clickable), the regular disabled state has the grey background